resolwe_bio/processes/support_processors/spikeins_qc.yml

```diff
@@ -0,0 +1,68 @@
+- slug: spikein-qc
+  name: Spike-ins quality control
+  requirements:
+    expression-engine: jinja
+    resources:
+      cores: 1
+    executor:
+      docker:
+        image: resolwebio/rnaseq:4.2.0
+  data_name: '{{ sample|sample_name|default("?") }} spike-ins QC'
+  version: 0.0.1
+  type: data:spikeins
+  category: other
+  persistence: CACHED
+  description: >
+    Plot spike-ins measured abundances for sample`s quality control. The process will output
+    graphs showing the correlation between known concentration of ERCC spike-ins and sample`s
+    measured abundance.
+  input:
+    - name: samples
+      label: List of samples
+      type: list:data:expression
+    - name: mix
+      label: Spike-ins mix
+      type: basic:string
+      choices:
+        - label: ERCC Mix 1
+          value: ercc_mix1
+        - label: ERCC Mix 2
+          value: ercc_mix2
+        - label: SIRV-Set 3
+          value: sirv_set3
+      description: >
+        Select spike-ins mix.
+  output:
+    - name: plots
```

Comment:
We may want to generalize this in case we include charts in the future that are not plots (for example a bar chart). For something to be a plot, both axes must be continuous.